pipelines/build/build_base_file.groovy

```diff
@@ -233,6 +233,24 @@ def createJob(jobName, jobFolder, config, enableTests, scmVars) {
     return create
 }
 
+def checkSaneConfig(releaseTag, jobConfigurations) {
+
+    if (releaseTag != null && releaseTag.length() > 0) {
```

Comment:
Does ` if (release tag) {` accomplish the same thing?

Reply:
just had a test and it appears so yes